src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/ElasticSearchRunConfiguration.java

```diff
@@ -32,7 +33,7 @@ public class ElasticSearchRunConfiguration implements Serializable {
 
     private static final String TIMESTAMP = "timestamp";
 
-    private static final DateTimeFormatter UTC_MILLIS = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSX");
+    private static final DateTimeFormatter UTC_NANOS = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSSSSSSSX");
```

Comment:
The name of the constant is misleading, because it does in now way enforce using UTC (the timezone is determined by the ZonedDateTime being formatted).

```suggestion
    private static final DateTimeFormatter TIMESTAMP_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSSSSSSSX");
```